.github/workflows/check.yml

```diff
@@ -0,0 +1,23 @@
+name: Check
+
+on:
+  push:
+    branches:
+      - gh-ci
```

Comment:
I presume this is temporary for now?

Reply:
Yes, it's temporary, it should be `master`.